controllers/imagerepository_controller.go

```diff
@@ -313,7 +313,11 @@ func (r *ImageRepositoryReconciler) ProvisionImageRepository(ctx context.Context
 			imageRepositoryName = imageRepository.Namespace + "/" + imageRepository.Name
 		}
 	} else {
-		imageRepositoryName = imageRepository.Namespace + "/" + imageRepository.Spec.Image.Name
+		if strings.HasPrefix(imageRepository.Spec.Image.Name, "/"+imageRepository.Namespace+"/") || strings.HasPrefix(imageRepository.Spec.Image.Name, imageRepository.Namespace+"/") {
```

Comment:
Can we trim `/` at the beginning of `imageRepository.Spec.Image.Name`, so the condition became more clear?

Reply:
ok